.cirrus.yml

```diff
@@ -50,6 +50,20 @@ task:
     - python3 -m pip install -r ci/run_tests/requirements.txt
     - python3 ci/run_tests/run_tests.py --interface en0 --use-sudo --test-args="-x TestPcapLiveDeviceSpecialCfg"
 
+task:
+  name: Regression Tests Linux (Memory Behavior Sanitizer)
+  container:
+    image: gcr.io/oss-fuzz-base/base-clang
+  configure_script:
+    - apt-get update && apt-get install -y libpcap-dev git make
+    - ./configure-fuzzing.sh --sanitizer memory
+    - ldconfig
+  build_script:
+    - export LIB_FUZZING_ENGINE="-fsanitize=fuzzer"
```

Comment:
@sashashura why is this line needed?

Reply:
It wasn't working without it. I think it is reading `LIB_FUZZING_ENGINE` from env somewhere. Let me quickly check.

Reply:
Makes sense, thanks for the explanation! 🙏